scripts/extension-registry-changed

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script keeps the list of extensions up to date 
+# in the docs/sources/next/extensions/explore.md file.
+#
+# The script is run by the extension-registry.changed.yml
+# workflow when the extension registry changes.
+# 
+# The list of extensions is generated based on https://registry.k6.io/registry.json.
+#
+# In the docs/sources/next/extensions/explore.md file 
+# the content of the <div class="nav-cards"> HTML element
+# is replaced with the generated extension list.
+
+generate_extension_list_partial() {
+curl -sL https://registry.k6.io/registry.json | 
+ jq -r '
+ map(select(.module != "go.k6.io/k6") | { name:.repo.name, url: .repo.url, description: .description } ) |
+ sort_by(.name) | map(
+"    <a href=\"\(.url)\" target=\"_blank\" class=\"nav-cards__item nav-cards__item--guide\">
+        <h4>\(.name)</h4>
+        <p>\(.description)</p>
+    </a>"
+) | .[] 
+'
+}
+
+replace_extension_list_partial() {
+ed -s $1 <<EOF
+/<div class=.nav-cards.>/+,/<.div>/-d
+/<div class=.nav-cards.>/ r $2
+w
+q
+EOF
+}
+
+scriptdir=$(dirname $(readlink -f "$0"))
+docfile=$scriptdir/../docs/sources/next/extensions/explore.md
```

Comment:
Quoting to avoid word splitting on paths with spaces

```suggestion
scriptdir="$(dirname "$(readlink -f "$0")")"
docfile="$scriptdir/../docs/sources/next/extensions/explore.md"
```